3/T2/index.js

```diff
@@ -0,0 +1,209 @@
+      alert('Sorry, but this sequence is not suitable for sorting dominoes');
+    }
+  } else {
+    alert('Sorry, but this sequence is not suitable for sorting dominoes');
+  }
+});
+
+buttonRandom.addEventListener('click', () => {
+  dominoUp.value = getRandomNumbers();
+  dominoDown.value = getRandomNumbers();
+  arrDominoUp = dominoUp.value.split('');
+  arrDominoDown = dominoDown.value.split('');
+  drawDomino(arrDominoUp, arrDominoDown);
+  resultEl.classList.add('is-none');
+});
+
+function addListener() {
+  const dominos = document.getElementsByClassName('item');
+  for (let i = 0; i < dominos.length; i++) {
+    dominos[i].addEventListener('click', changeDomino);
+  }
+}
+
+function findAppropriateNumber(str) {
+  // Here I find the number of numbers that are greater than or equal to rangeNumbers.maxLength
+  const arrDomino = str.split('');
+  for (let i = 1; i < arrDominoUp.length + 1; i++) {
+    const newArr = arrDomino.filter( elem => {
+      return +elem === i;
+    });
+    if (newArr.length >= arrDominoUp.length) {
+      return { 
+        number: newArr[0], 
+        quantity: newArr.length 
+      };
+    }
+  }
+  return { 
+    number: 0, 
+    quantity: 0 
+  };
+}
+
+function сheckOnQuantityAppropriateNumber(obj) {
+  //Here I am looking for the number of equal numbers of dominoes top and bottom, example 3/3
+  let number = 0;
+  let resultQuantityAppropriate = 0;
+  const rightAmount = obj.quantity - arrDominoUp.length;
+  for (let i = 0; i < arrDominoUp.length; i++) {
+    if (+arrDominoUp[i] === +arrDominoDown[i]) {
+      resultQuantityAppropriate++;
+    }
+  }
+  if (resultQuantityAppropriate <= rightAmount) {
+    number = obj.number;
+  }
+  return number;
+}
+
+function rotateSomeNumbers(number) {
+  const objChange = findChange(number);
+  if (objChange.quantityСhange > arrDominoUp.length / 2) {
+    const arrAllNumbers = createArr(rangeNumbers.max);
+    objChange.arrNumberLineDominoСhange = arrAllNumbers.filter( el => !objChange.arrNumberLineDominoСhange.includes(el));
+    objChange.quantityСhange = arrDominoUp.length - objChange.quantityСhange;
+    return {
+      topArr: arrDominoDown,
+      bottomArr: arrDominoUp,
+      line: objChange.arrNumberLineDominoСhange,
+      change: objChange.quantityСhange,
+    };
+  }
+  return {
+    topArr: arrDominoUp,
+    bottomArr: arrDominoDown,
+    line: objChange.arrNumberLineDominoСhange,
+    change: objChange.quantityСhange,
+  };
+}
+
+function findChange(number) {
+  const arrNumberLineDominoСhange = [];
+  let quantityСhange = 0;
+  const cloneArr = arrDominoUp.concat();
+  cloneArr.forEach((num, i) => {
+    const numberArrDominoDown = arrDominoDown[i];
+    if (!(+num === +number)) {
+      arrDominoUp[i] = numberArrDominoDown;
+      arrDominoDown[i] = num;
+      arrNumberLineDominoСhange.push(i + 1);
+      quantityСhange++;
+    }
+  });
+  return {
+    arrNumberLineDominoСhange: arrNumberLineDominoСhange, 
+    quantityСhange: quantityСhange
```

Comment:
do not forget comma in the end of last key: value - this is good for git, if you add one more key-value, you will changes only 1 line. When no comma on last line - you'll see changes in two lines when pushing to git.